Shared/API/Onboarding+Cloud.swift

```diff
@@ -76,36 +76,15 @@ struct VerifyCode: CloudFunction {
 struct ActivateUser: CloudFunction {
 
     typealias ReturnType = Any
-    let fullName: String
```

Comment:
I'm decoupling activating the user from entering the name. We can save the save the user's name immediately after they enter it.

Reply:
I don't think so, but confirm with @gcolotti